src/types/api.js

```diff
@@ -73,6 +73,34 @@
  * @prop {boolean} [$anchorTimeout]
  */
 
+/**
+ * @typedef AnnotationData
+ * @prop {string} uri
+ * @prop {string} $tag
+ * @prop {string} [$highlight]
+ * @prop {SelectSource[]} target
```

Comment:
This is the same as the `target` property of the `Annotation` type above isn't it. Perhaps unify that and `SelectSource` into a single `Target` definition? The `target` property would then be a `Target[]`.

Reply:
Is the same true for `document` between AnnotationData.document vs Annotation.document.

Reply:
bah! yes, missed that.